config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -579,6 +579,8 @@ presubmits:
           value: "win2019"
         - name: PREPULL_YAML
           value: "prepull-head.yaml"
+        - name: KUBE_RELEASE_RUN_TESTS
```

Comment:
please stop running this with `--build=release` in presubmit, it's extremely expensive and slow:  https://github.com/kubernetes/test-infra/issues/18943